### PR TITLE
[FEATURE] Select preselect

### PR DESCRIPTION
**Pull request by turbo-ele**

### CODE REVIEW

File: dist/husky.js [hunk 1]
```diff
@@ -38518,10 +38520,12 @@ define('__component__$select@husky',[], function() {
             // Used as a fallback to revert to the last committed data
             this.mergedData = null;
 
-            // when preselected elements is not set via options look in data-attribute
-            if (!this.options.preSelectedElements || this.options.preSelectedElements.length === 0) {
+            // if data value is written,
+            if (!this.options.preSelectedElements ||
+                this.options.preSelectedElements.length === 0 ||
+                this.options.preferPreselectionFromData
```
Comment (wachterjohannes) [inline dist/husky.js]: why do you need that option?

**Comment by turbo-ele** [comment]
add default option

File: husky_components/select/main.js [hunk 2]
```diff
@@ -74,7 +75,8 @@ define([], function() {
             resultKey: '',
             showToolTip: false,
             translations: translations,
-            isNative: false
+            isNative: false,
+            default: null
```
Comment (wachterjohannes) [inline husky_components/select/main.js]: default is a keyword use `defaultValue` instead ...

**Comment by wachterjohannes** [comment]
else ACK

**Pull request merged**